tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,39}-{vanilla,pyopenssl}
+envlist = py{2.7,3.10}-{vanilla,pyopenssl}
```

@strokirk I think pyenv isn't finding the environments by that name. It's looking for something that fits the ["pyNM" pattern](https://tox.wiki/en/latest/example/basic.html#a-simple-tox-ini-default-environments). Obviously "py310" doesn't match exactly, but it might be worth a try?
```suggestion
envlist = py{27,310}-{vanilla,pyopenssl}
```